modules/apps/josh-lapurd-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir='<@liferay.language key="lang.dir" />' lang="${w3c_language_id}">
+
+<head>
+	<title>${the_title} - ${company_name}</title>
+
+	<meta content="initial-scale=1.0, width=device-width" name="viewport" />
+
+	<#if is_signed_in>
+		<style>
+			#jw-banner {
```

Comment:
We should have generic CSS classes and Id's but have them relate and be specific to what they target.